Plots/Bar/NCL_bar_11.py

```diff
@@ -40,7 +39,7 @@
 panel = 0
 for row in range(0, 2):
     for col in range(0, 2):
-        # Use geocat.viz.util convenience function to set axes parameters
+        # Use geocat.viz.util convenience function to set axis parameters
```

Comment:
```suggestion
        # Use geocat.viz.util convenience function to set axes parameters
```
This function is setting the parameters for both axes at once, so 'axes' would be correct here